pkg/authorization/authorizer/scope/converter.go

```diff
@@ -154,15 +176,15 @@ func (userEvaluator) Validate(scope string) error {
 func (userEvaluator) Describe(scope string) (string, string, error) {
 	switch scope {
 	case UserInfo:
-		return "Read-only access to your user information (including username, identities, and group membership)", "", nil
+		return defaultSupportedScopesMap[UserInfo], "", nil
```

Comment:
Collapse the first four cases since they are the same and use `defaultSupportedScopesMap[scope]` for all cases.